modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc

```diff
@@ -82,72 +81,76 @@ This table shows the hardware requirements for deploying {productname} Proxy.
 
 |===
 
+// FIXME: ^^^ should we better just mention
+//            /var/lib/containers/storage/volumes here above? ^^^
```

Comment:
Paths above are wrong. I though we already fixed those.

We should not mention individual volumes because in fact only one volume is used, the rest are basically empty or for config files.
But more importantly `mgr-storage-proxy` works only over whole volume directory.

So please remove any mentioned of `[path]``/var/lib/containers/storage/volumes/srv-www`` ` and `[path]``/var/lib/containers/storage/volumes/var-cache`` ` and replace it with only one path `[path]``/var/lib/containers/storage/volumes`` ` which should be 100GB minimum.

Reply:
Yes, but I missed that here.  I only fixed the requirements in `installation-and-upgrade/pages/hardware-requirements.adoc`.  I unfortunately do not know where we book-people to have the requirements documented...

If contents in `installation-and-upgrade/pages/hardware-requirements.adoc` is also good enough for the vm section for proxy deployments, I'd just reference that reqs file and remove it from the deployment files.

Reply:
BTW, I created a new section for proxy in `persistent-container-volumes.adoc`.  Does it make sense?  If yes, can you please replace `???` with actual directories from inside the container?